src/map/ai/ai_mob_dummy.cpp

```diff
@@ -719,6 +719,7 @@ void CAIMobDummy::ActionAbilityStart()
     // We don't have any skills we can use, so let's go back to attacking
     if (MobSkills.size() == 0)
     {
+        m_PMob->PBattleAI->SetMobAbilityEnabled(false);
         TransitionBack(true);

```

Comment:
> This would break mob's that don't have any skills and then later on get skills.

What are you smoking?

If you add skills to mobs that don't have any you are restarting so it can see the SQL change anyways, at which point this no longer effects that mob.